epoxy-integrationtest/src/test/java/com/airbnb/epoxy/ControllerLifecycleHelper.java

```diff
@@ -36,11 +39,16 @@ public View bindModel(EpoxyModel<?> model) {
   }
 
   static EpoxyViewHolder createViewHolder(BaseEpoxyAdapter adapter, int position) {
-    int itemViewType = adapter.getItemViewType(position);
-    return adapter.onCreateViewHolder(
-        new FrameLayout(RuntimeEnvironment.application),
-        itemViewType
+    final EpoxyViewHolder viewHolder = spy(
+        adapter.onCreateViewHolder(
+            new FrameLayout(RuntimeEnvironment.application),
+            adapter.getItemViewType(position)
+        )
     );
+
+    // The simplest way to inject the position for testing.
+    when(viewHolder.getAdapterPosition()).thenReturn(position);
```

Comment:
Although I feel this isn't ideal, I found it to be the least obtrusive. Would love feedback if anyone has a more proper idea on how to fix.

Reply:
This seems reasonable, thanks for fixing the tests too 👍